corehq/apps/hqwebapp/forms.py

```diff
@@ -21,6 +24,23 @@ def clean_username(self):
         username = self.cleaned_data['username'].lower()
         return username
 
+    def clean(self):
+        lockout_message = mark_safe(_('Sorry - you have attempted to login with an incorrect password too many times. Please <a href="/accounts/password_reset_email/">click here</a> to reset your password.'))
```

Comment:
Should we be vague and say 'incorrect credentials' instead of 'incorrect password' so as not to tell a malicious agent whether a username is correct or incorrect?